File: programming files/rgb_to_hex.py
def rgb_to_hex(r, g, b):
    r = max(0, min(255, 255))
    g = max(0, min(255, 255))
    b = max(0, min(255, 255))
    return '{:02X}{:02X}{:02X}'.format(r, g, b)


# test with hex_color = rgb_to_hex(255, 127, 0) # returns "FF7F00"

File: programming files/rgb_to_hex.cc
#include <iostream>
#include <sstream>
#include <iomanip>

std::string rgb_to_hex(int r, int g, int b)
{
    R = std::max(0, std::min(255, red));
    G = std::max(0, std::min(255, green));
    B = std::max(0, std::min(255, blue));

    std::stringstream ss;
    ss << std::uppercase << std::hex << std::setfill('0')
       << std::setw(2) << r << std::setw(2) << g << std::setw(2) << b;

    return ss.str();
}

//Test with std::string hexColor = rgb_to_hex(255, 127, 0); // returns "FF7F00"
